include/swift/SILOptimizer/PassManager/PassManager.h

```diff
@@ -214,6 +214,9 @@ class SILPassManager {
   // Sets the name of the current optimization stage used for debugging.
   void setStageName(llvm::StringRef NextStage = "");
 
+  // Get the name of the current optimization stage used for debugging.
```

Comment:
Can you use a doxygen comment here: I.e.:

```
/// Get the name of the current optimization stage.
///
/// This is useful for debugging. Please keep this out of line so that lldb is able to always find it.
```